src/common/w25x_communication.cpp

```diff
@@ -49,16 +52,20 @@ enum {
     EVENT_TX_COMPLETE = (1 << 1),
 };
 
-extern "C" bool w25x_communication_init() {
-    // clear error
-    current_error = 0;
-
+extern "C" bool w25x_communication_init(bool init_event_group) {
     // check SPI handle has been assigned using w25x_spi_assign
     if (spi_handle == NULL)
         return false;
 
+    HAL_SPI_Abort(spi_handle);
+    w25x_cs_high();
+    DELAY_NS(50);
```

Comment:
Please add a documentation what are we waiting for here and why 50 ns

Reply:
documented